documentation/milestones/individual_milestones/get_opportunities.md

```diff
@@ -0,0 +1,196 @@
+| Document Status | Accepted                                                             |
+| Epic Link       | [Issue 70](https://github.com/HHS/grants-equity/issues/70)           |
+| Epic Dashboard  | [Milestone Roadmap](https://github.com/orgs/HHS/projects/12/views/4) |
+| Target Release  | 2023-10-11                                                           |
+| Product Owner   | Lucas Brown                                                          |
+| Document Owner  | Billy Daly                                                           |
+| Lead Developer  | Aaron Couch                                                          |
+| Lead Designer   | Andy Cochran                                                         |
+
+## Short description
+<!-- Required -->
+
+Deploy a public API endpoint to PROD that allows users to see at least one field per grant opportunity listed in grants.gov
+
+## Goals
+
+### Business description & value
+<!-- Required -->
+
+The launch of a public API endpoint which provides information about every grant opportunity in grants.gov represents the culmination of multiple internally focused deliverables and serves as the foundation for future development that relies on the API.
+
+By delivering this public endpoint and ensuring it remains available even when the legacy grants.gov site is experiencing planned or unplanned outages, this milestone aims to demonstrate the following value propositions:
+
+- Exposes `beta.grants.gov/api/` as the public API for the project that technical stakeholders can access and where future endpoints and/or features will be released
+- Proves the successful completion of technical milestones that enable faster development on the API without sacrificing code quality or security
+- Delivering another public win that both internal and external stakeholders can rally around, which helps build momentum and enthusiam for the project
+
+### User Stories
+<!-- Required -->
+
+- As an **HHS staff member**, I want:
+  - the API to adopt the proper security practices, so that we have a strategy for preventing and responding to security vulnerabilities before the API is launched
+  - published data about opportunities to be consistent between legacy grants.gov and `beta.grants.gov`, so that users won't be confused by discrepancies between these sources
+- As a **consumer of the API**, I want:
+  - clear documentation and a user guide for the API, so that I don't have to rely on reading the source code to learn how to interact with and consume from it
+  - changes made to a given endpoint to be backward-compatible, so that I can start building against this API without worrying about breaking changes
+- As a **project maintainer**, I want:
+  - most of the critical development infrastructure to be in place when we officially launch the API, so that we can deploy bug fixes or new features quickly once the site is live
```

Comment:
Suggest something like: 

- the API code to be designed in such a way to minimize repetitive configuration for data model (i.e., to update the definition of a field in the API, the same information does not need to be manually added to the DB migration scripts, the core data class, the API definition, and the documentation, but rather all these things can be updated automatically with only one change that propagates throughout)

Reply:
Added in [this commit](https://github.com/HHS/grants-equity/pull/249/commits/3aa14fa418605012f4e9766952271ca22b97e8f2)